src/197F40.c

```diff
@@ -2260,7 +2260,22 @@ ApiStatus GetPlayerActorID(Evt* script, s32 isInitialCall) {
 
 INCLUDE_ASM(s32, "197F40", func_8026E9A0);
 
-INCLUDE_ASM(s32, "197F40", GetDistanceToGoal);
+ApiStatus GetDistanceToGoal(Evt* script, s32 isInitialCall) {
+    s32* args = script->ptrReadPos;
+    s32 actorID = evt_get_variable(script, *args++);
+    s32 outVar = *args;
```

Comment:
Please use `*args++` if it matches - We always want to advance to the next possible argument, even if it is not used afterwards.